libcontainer/rootfs_linux.go

```diff
@@ -180,18 +180,22 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 			if err != nil {
 				return err
 			}
+			relDir, err := filepath.Rel(mm.Root, dir)
```

Comment:
I tried to print stuff and it looks exactly like I described. Mountpoints in container somehow became pretty weird, like:
```/system.slice/docker-e36b665d5dfaf86221d5b0edba3ad7bfe3f7e0af45aa78085aa1f7795706c5fd.scope/system.slice/docker-e36b665d5dfaf86221d5b0edba3ad7bfe3f7e0af45aa78085aa1f7795706c5fd.s
cope/docker/8cccc84d71a08a64948c297f7117b777e69fc337b75f2df3fa78c1515f908933
```
Maybe I did logical mistake somewhere, or maybe it is expected.